addon/adapters/m3-debug-adapter.js

```diff
@@ -33,6 +33,7 @@ export default class M3DebugAdapter extends DataAdapter {
     @return Object with stringified values as needed
   */
   _stringifyNestedValues(nestedJSON) {
```

Comment:
Let's update this function's description since the behavior is probably counter-intuitive to an average reader. Also can you add a quick test? Can these steps be simulated by an integration test possibly (not sure tbh)? Thanks!

Reply:
Sorry @SYU15 been a busy week, I just updated the comment, and I'm trying to create some integration tests, somehow I'm unable to replicate it on tests, gonna dig a little bit more I think, I'll keep you posted